release/cc/execute_targets.py

```diff
@@ -83,16 +83,22 @@ def execute_and_extract_configs(container_configs_list, bazel_version):
       # Generate config directory.
       os.makedirs(config.get_config_dir())
 
-      command = ("bazel run --define=DOCKER_AUTOCONF_OUTPUT={OUTPUT_DIR} "
-                 "//{PACKAGE}:{TARGET}").format(
+      command = ("bazel build //{PACKAGE}:{TARGET}").format(
+                     PACKAGE=container_configs.package,
+                     TARGET=target)
+      print("\nExecuting command: %s\n" % command)
+      subprocess.check_call(shlex.split(command))
+
+      command = ("cp ./bazel-out/k8-fastbuild/bin/{PACKAGE}/{TARGET}_outputs.tar "
```

Comment:
Will this make the tarball left in the {OUTPUT_DIR}, if so, we should delete it after extraction.

Reply:
the existing functionality was to output the tar to the OUTPUT_DIR
I'm just keeping it on par by copying it to OUTPUT_DIR after bazel build
 (i.e., this should already be cleaned up by the existing clean up process)